procedure_branch

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+. settings
+
+TARGET_DATE="$1"
+OWNER="$2"
+ENGINEER="$3"
+
+if [[ -z $TARGET_DATE ]] ; then
+	read -r -p "Target date (YYYY-MM-DD): " TARGET_DATE
+	READ=1
+fi
+if [[ -z $OWNER ]] ; then
+	read -r -p "Release owner (Discourse usernmame): " OWNER
+	READ=1
+fi
+if [[ -z $ENGINEER ]] ; then
+	read -r -p "Release engineer (Discourse usernmame): " ENGINEER
+	READ=1
+fi
+if [[ $READ == 1 ]] ; then
+	echo >&2
+fi
+
+./ensure_clean_git_checkouts tool_belt > /dev/null
+# TODO: ensure checkout is up to date?
+
+"${GIT_DIR}/tool_belt/tools.rb" procedure branch "$PROJECT" "$VERSION" "$TARGET_DATE" "$OWNER" "$ENGINEER"
```

Comment:
does that work? I would have expected a `bundle install` somewhere inbetween

Reply:
I kind of debated it. Technically you need clamp and that's sufficient, which avoids a lot of issues with bundler.

I've been thinking about moving all the procedure stuff into this repository and trim it down to just native Ruby.

Reply:
https://github.com/theforeman/tool_belt/pull/530 would be that standalone command. I'll first move forward with that, and later consider moving over the procedures themselves.